feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/PreviewScreenComponents.kt

```diff
@@ -60,8 +65,27 @@ import kotlinx.coroutines.CompletableDeferred
 
 private const val TAG = "PreviewScreen"
 
-/** this is the preview surface display. This view implements gestures tap to focus, pinch to zoom,
- * and double tap to flip camera */
+
+/**
+ * Displays a [Toast] with specifications set by a [ToastMessage].
+ *
+ * @param toastMessage the specifications for the [Toast].
+ * @param onToastShown called once the Toast has been displayed.
+ */
+@Composable
+fun ShowToast(modifier: Modifier = Modifier, toastMessage: ToastMessage, onToastShown : () -> Unit){
+    Box(
+        modifier.testTag(toastMessage.testTag).semantics { contentDescription = toastMessage.testDesc }
+    ) {
+        Toast.makeText(LocalContext.current, toastMessage.message, toastMessage.toastLength).show()
+        onToastShown()
+    }
```

Comment:
I think this is just creating an empty `Box`, since Toasts aren't Composables. That empty box will get removed almost immediately since `onToastShown` will then cause recomposition to no longer draw the Box.  But maybe that's sufficient for UiAutomator to catch it?

Just checking if that's the intended behavior.

Reply:
yeah the intention was creating an "invisible box" to both be the vehicle to launch a toast message and also to identify its presence (with the tags). 

I'm not 100% sure if the UiAutomator will be able to catch it because of how fast it is, but i could include a delay  before it calls `ShowToast()` if it ends up being necessary to detect

Reply:
Since you're not 100% sure, it would be good to add a simple UiAutomator test for the `ShowToast` composable. Actually it's a good idea to add a test even if you are 100% sure, but especially if you have doubts.